src/frontend/js/index.js

```diff
@@ -96,7 +98,17 @@ const nerf = x => (1+erf(x*2-1))/2
 
 // get data from Julia:
 const getData = async endpoint =>
-    (await fetch("/api/" + endpoint)).json().catch(e => console.error(`Error getting data from:\n/api/${endpoint}\n\n`, e))
+    (await fetch("/api/" + endpoint + (endpoint.includes("?") ? "&" : "?") + "v=" + (await getApiVersion()))).json().catch(e => console.error(`Error getting data from:\n/api/${endpoint}\n\n`, e))
```

Comment:
This looks like a mistake? You do .json() in getApiVersion, so you shouldn't need to do it here as well.

Reply:
It's `fetch(...).json()` : )